easybuild/easyblocks/c/cuda.py

```diff
@@ -114,3 +148,12 @@ def make_module_req_guess(self):
         })
 
         return guesses
+    
+    def make_module_extra(self):
+        """Specify CUDA_HOME """
+
+        txt = super(EB_CUDA, self).make_module_extra()
+
+        txt += self.module_generator.set_environment('CUDA_HOME', self.installdir)
```

Comment:
Correct, but for some reason it is not working. We should keep the discussion here, instead of the one in https://github.com/hpcugent/easybuild-easyconfigs/pull/2177 since this is an easyblock issue. In any case, why use guesses instead of setting these PATHs manually to well known directories?